src/gpu/SDL_sysgpu.h

```diff
@@ -373,6 +373,71 @@ static inline bool IsIntegerFormat(
     }
 }
 
+static inline bool IsCompressedFormat(
```

Comment:
Comment for another PR: Can we move these inline functions into SDL_gpu.c? It makes things hard to find in this file and none of them are performance critical.